docs/topics/jobs.rst

```diff
@@ -96,4 +96,8 @@ But this will::
         somearg = response.meta['somearg']
         print "the argument passed is:", somearg
 
+If you wish to log the requests that couldn't be serialized, you can set the
+``SCHEDULER_DEBUG`` setting to ``True`` in the project's settings page.
```

Comment:
You can add a direct link to the setting:

    you can set the :setting:`SCHEDULER_DEBUG` to ...